docs/Language-specification.md

```diff
@@ -173,14 +173,14 @@ Adjacent string forms or variables are treated as a single leaf and concatenated
 
 **Examples:**
 ```text
-path/with/'Weird $Directory'/file.cpp    => path/with/Weird $Directory/file.cp
+path/with/'Weird $Directory'/file.cpp    => path/with/Weird $Directory/file.cpp
 "combining "'forms 'in-any" which way"   => combining forms in-any which way
 ```
 
 ### Bracket expressions
 Bracket expressions have the following form (literal `[]`, not optional):
 ```text
-[ <rule_invocation>|on <target> <rule_invocation> ]
+[ <rule_invocation>|on <target> <value_invocation> ]
```

Comment:
`value_invocation` was a leftover from a previous version of the grammar; I'll fix this part to be correct.